torch/csrc/distributed/rpc/BuiltinOp.cpp

```diff
@@ -0,0 +1,73 @@
+#include <torch/csrc/distributed/rpc/BuiltinOp.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+BuiltinOp::BuiltinOp(
+    std::shared_ptr<Operator> op, std::vector<at::IValue>&& args)
+    : op_(std::move(op)), stack_(args) {}
+
+BuiltinOp::~BuiltinOp() = default;
```

Comment:
nit: If you're just going to declare the destructor as default, you don't need to declare it at all.